modules/v1_16_R2/src/main/java/de/Keyle/MyPet/compat/v1_16_R2/entity/CraftMyPet.java

```diff
@@ -68,6 +68,15 @@ public boolean isSitting() {
         return getHandle().isSitting();
     }
 
+    @Override
+    public boolean isInvisible() {
+        return getHandle().isInvisible();
+    }
+
+    @Override
+    public void setInvisible(boolean invisible) {
+    }
+
```

Comment:
Could you put that into an extra PR please

Reply:
https://github.com/xXKeyleXx/MyPet/pull/1418